go-controller/pkg/node/node.go

```diff
@@ -138,9 +138,16 @@ func isOVNControllerReady(name string) (bool, error) {
 	err = wait.PollImmediate(500*time.Millisecond, 60*time.Second, func() (bool, error) {
 		stdout, _, err := util.RunOVSOfctl("dump-aggregate", "br-int")
 		if err != nil {
+			klog.V(5).Infof("Error dumping aggregate flows: %v "+
+				"for node: %s", err, name)
 			return false, nil
 		}
-		return !strings.Contains(stdout, "flow_count=0"), nil
+		ret := strings.Contains(stdout, "flow_count=0")
+		if !ret {
+			klog.V(5).Infof("Got a flow count of 0 when "+
+				"dumping flows for node: %s", name)
+		}
+		return !ret, nil
```

Comment:
Clarification : Should this also be `ret` instead of `!ret`

Reply:
No, we want to keep returning false if the flow count is indeed 0. for anything else we want to return true and bail out of the wait(). Check for documentation on `wait.PollImmediate`

Reply:
Ok, Thanks for clarifying.